### PR TITLE
[PHP] khan.php: Improve ident

### PR DESCRIPTION
**Pull request by BaseMax**
-  Improve ident tab

### CODE REVIEW

File: scrapers/php/khan.php [hunk 1]
```diff
@@ -1,13 +1,11 @@
 <?php
+include('base/common.php');
```
[the remark this answers is not in the document]
Comment (BaseMax) [reply scrapers/php/khan.php]: Done, only need doc comment.

**Comment by BaseMax** [comment]
At next, I invite someone who want to help improve and complete Doc comment of the file.

**Review by jbampton — COMMENTED** [review]
Pretty sure this doesn't work anymore since the whole site has changed.

@ajakov was working on a similar PR that he closed.

**Comment by ajakov** [comment]
@jbampton I think this is related to my PR - auto generated by the Hound.

**Comment by jbampton** [comment]
@ajakov yes I know the Hound bot is savage :)

**Comment by jbampton** [comment]
https://houndci.com/

**Review by jbampton — APPROVED** [review]

**Pull request merged**